# Parameters for the simulation
sim:
  propogate_rate: 200.0   # How often the dynamics are updated in Hz
